testing/matrix/linter/rules/resources/pdb.go

```diff
@@ -55,11 +55,40 @@ func ControllerMustHavePDB(linter *rules.ObjectLinter) {
 			continue
 		}
 
+		if isPodControllerDaemonSet(object.Unstructured.GetKind()) {
+			continue
+		}
+
 		lerr := ensurePDBIsPresent(pdbSelectors, object)
 		linter.ErrorsList.Add(lerr)
 	}
 }
 
+// ControllerMustNotHavePDB adds linting errors if there are pods from DaemonSets which are covered
+// by a PodDisruptionBudget
+func ControllerMustNotHavePDB(linter *rules.ObjectLinter) {
```

Comment:
The name can be more specific, e.g. `DaemonSetMustNotHavePDB`

Reply:
good point, thanks